frontend/dockerfile/cmd/dockerfile-frontend/hack/release

```diff
@@ -75,6 +75,18 @@ if [[ "$RELEASE" = "true" ]] && [[ "$GITHUB_ACTIONS" = "true" ]]; then
   nocacheFilterFlag="--no-cache-filter=base"
 fi
 
+outputFlag="type=image,$pushFlag"
+if [ "$PUSH" = "push" ] && [ "$GITHUB_ACTIONS" = "true" ]; then
```

Comment:
Like the BuildKit image https://github.com/moby/buildkit/actions/runs/10152000989/job/28073017223?pr=5197#step:7:24 we should also generate annotations on PR, not just push.

```suggestion
if [ "$GITHUB_ACTIONS" = "true" ]; then
```